frameworks/pytorch/get_started_mnist_deploy.ipynb

```diff
@@ -6,23 +6,15 @@
    "source": [
```

Comment:
<p>Grammar (pre-existing): <code>Local mode are typically used</code> -&gt; <code>Local mode is typically used</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>